pkg/imgpkg/bundle/bundle.go

```diff
@@ -30,7 +32,7 @@ type Bundle struct {
 }
 
 func NewBundle(ref string, imagesMetadata ctlimg.ImagesMetadata) *Bundle {
-	return NewBundleWithReader(ref, imagesMetadata, &singleLayerReader{})
+	return &Bundle{plainimg.NewPlainImage(ref, imagesMetadata), imagesMetadata, &singleLayerReader{}}
```

Comment:
Any particular reason not to use the `NewBundleWithReader`
 constructor?

Reply:
Nope. Changed!